lib/lookup.json

```
@@ -345,6 +345,11 @@
     "maintainers": "kriskowal",
     "skip": ["win32", ">=21"]
   },
+  "readable-stream": {
+    "prefix": "v",
+    "maintainers": "nodejs/streams",
+    "skip": ["win32", "ppc", "aix"]
```

```suggestion
    "skip": ["win32", "aix"]
```

[Passes on Linux on ppc64le](https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3363/nodes=rhel8-ppc64le/testReport/(root)/citgm/readable_stream_v4_5_0/), so no need to skip for "ppc".